### PR TITLE
Fixes for Android Q

### LINKED ISSUE #16: No audio on Pixel 3
Have an issue where I can hear no audio on my pixel 3. I have the ability to control all my other clients but the pixel 3 does not show up as one of the devices but the Snapclient is showing running.... when I hit the play button.

### PR DESCRIPTION
**Pull request by derWeihnachtsmann**
Binary cannot be loaded from "external" anymore.. :( 
fixes #16

### CODE REVIEW

**Comment by badaix** [comment]
where are the binaries coming from?

**Comment by derWeihnachtsmann** [comment]
> 
> 
> where are the binaries coming from?

I just copied them from /src/main/assets/bin.

**Comment by badaix** [comment]
Thanks,

renaming the binaries to libXXX.so is ugly! 
But I googled and on [reddit](https://www.reddit.com/r/androiddev/comments/b2inbu/psa_android_q_blocks_executing_binaries_in_your/) they came up with the same workaround.

This means that we won't need this copy and setup stuff anymore?

In MainActivity.java
```java
        new Thread(new Runnable() {
            @Override
            public void run() {
                Log.d(TAG, "copying snapclient");
                Setup.copyBinAsset(MainActivity.this, "snapclient", "snapclient");
                Log.d(TAG, "done copying snapclient");
            }
        }).start();
```
And all of those `CopyAsset` and `CopyBinAsset` in `Setup.java`?

**Comment by badaix** [comment]
After following GitHub's manual merge instructions because of a conflict, I get:
`Unsupported Modules Detected: Compilation is not supported for following modules: libs, libsnapcast. Unfortunately you can't have non-Gradle Java modules and Android-Gradle modules in one project.`

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Snapcast/Snapcast.iml" filepath="$PROJECT_DIR$/Snapcast/Snapcast.iml" />
+      <module fileurl="file://$PROJECT_DIR$/libs/libs.iml" filepath="$PROJECT_DIR$/libs/libs.iml" />
```
Comment (badaix) [inline .idea/modules.xml]: this file seems to be missing

File: .idea/modules.xml [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
+      <module fileurl="file://$PROJECT_DIR$/libs/libsnapcast/libsnapcast.iml" filepath="$PROJECT_DIR$/libs/libsnapcast/libsnapcast.iml" />
```
Comment (badaix) [inline .idea/modules.xml]: and this as well

File: settings.gradle [hunk 3]
```diff
@@ -1 +1,2 @@
 include ':Snapcast'
+include ':libs:libsnapcast'
```
Comment (badaix) [inline settings.gradle]: Gradle is complaining about this: `Unsupported Modules Detected: Compilation is not supported for following modules: libs, libsnapcast. Unfortunately you can't have non-Gradle Java modules and Android-Gradle modules in one project.`

**Comment by badaix** [comment]
@derWeihnachtsmann thanks, it's now merged and builds. Because of my formerly unpushed changes, there was a merge conflict, and now you are for some reason not listed as contributor. Sorry for that, I don't know how to fix this.

**Pull request closed without merging**